annotation/copy_test.go

```diff
@@ -0,0 +1,121 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package annotation
+
```

Comment:
A continuation of the comment made to previous PR: after seeing this, I think we might be more incentivized to move the test utility stuff to a separate package (say, `internal/nilawaytest`) for easier maintenance.

Reply:
Created issue #172 to perform this refactoring in a follow-up PR.